### DISCUSSION TITLE
v0.2.4

### DISCUSSION THREAD

**Discussion by RobertCraigie** [RobertCraigie/prisma-client-py#86] [General]
This release is a patch release, the v0.2.3 release erroneously contained auto-generated files.

<hr /><em>This discussion was created from the release <a href='https://github.com/RobertCraigie/prisma-client-py/releases/tag/v0.2.4'>v0.2.4</a>.</em>

**Comment by RobertCraigie** [comment]
@kafai97 Fixed the release.

I have added steps to my package publishing workflow to prevent the issue from happening again.

Thank you for reporting the issue!

**Reply by kafai-lam** [reply]
Thank you for creating this incredible package by yourself! Appreciate your effort!

**Reply by RobertCraigie** [reply]
Thank you!

Do you have any suggestions for improvements I could make to the client?
